victims/factory/daemonsets/daemonsets.go

```diff
@@ -0,0 +1,63 @@
+package daemonsets
```

Comment:
Don't bundle two PRs into one. 
Each PR should not build upon the other. This PR should be isolated to fixing to reloading the config map.